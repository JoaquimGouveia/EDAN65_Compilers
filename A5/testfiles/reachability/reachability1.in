int main(){
    print(1);
    return f();
}

int f() {
    return g();
}

int g() {
    return h();
}

int h() {
    return 2;
}