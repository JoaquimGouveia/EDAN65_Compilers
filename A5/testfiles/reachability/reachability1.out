Function: main
f
h
g
print
---------------
Function: f
h
g
---------------
Function: g
h
---------------
Function: h
---------------
