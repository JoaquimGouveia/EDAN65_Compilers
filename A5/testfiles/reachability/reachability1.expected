Function: main
h
f
g
print
---------------
Function: f
h
g
---------------
Function: g
h
---------------
Function: h
---------------
